cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -33,6 +33,12 @@ private class PrimaryArgumentNode extends ArgumentNode {
     or
     op = call.getThisArgumentOperand() and pos = -1
   }
+
+  override string toString() {
+    result = "Argument " + op.(PositionalArgumentOperand).getIndex()
+    or
+    op instanceof ThisArgumentOperand and result = "this"
```

Comment:
```suggestion
    op instanceof ThisArgumentOperand and result = "Call qualifier"
```

or

```suggestion
    op instanceof ThisArgumentOperand and result = "This argument"
```

or something like that. Just like we distinguish parameters from arguments, we have to distinguish `this` from what we call the _qualifier_ in QL.

Reply:
Good call (pun slightly intended). I chose the second option to stay consistent with the other case, but I'm open to alternatives if anyone prefers something else in the future.